src/plugins/oauth2/openIdConfiguration.ts

```diff
@@ -1,26 +1,63 @@
 import { log, userInteractionProvider } from '../../io';
 import type * as models from '../../models';
 import * as utils from '../../utils';
+import { isEqualWith } from 'lodash';
 import get from 'lodash/get';
 import { URL } from 'url';
 
 export const DEFAULT_CALLBACK_URI = 'http://localhost:3000/callback';
 const DefaultOAuthVariablePrefix = 'oauth2';
 
-function getVariable(variables: models.Variables, variablePrefix: string | undefined, name: string): string {
-  const getter = (prefix: string) => variables[`${prefix}_${name}`] || get(variables, `${prefix}.${name}`);
+function getVariableRaw(variables: models.Variables, name: string, variablePrefix: string): unknown {
+  let varValue = variables[`${variablePrefix}_${name}`];
+  if (typeof varValue === 'undefined' || varValue === null) {
+    varValue = get(variables, `${variablePrefix}.${name}`);
+  }
+  return varValue;
+}
+
+function getVariableUnknown(variables: models.Variables, variablePrefix: string | undefined, name: string): unknown {
+  const getter = getVariableRaw.bind(undefined, variables, name);
```

Comment:
just a question: I hardly ever use bind. I would not have used `undefined` here, but probably `this`. Is there an advantage for `undefined`?

Reply:
IIRC, TSC flagged `this`. The `this` really does not matter here anyways, I could've just made two regular function calls instead.